ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -66,7 +66,8 @@ export class Pay extends IronfishCommand {
     const expirationSequence = flags.expirationSequence
     const memo = flags.memo || ''
 
-    const client = await this.sdk.connectRpc()
```

Comment:
These could also change to `const client = await this.sdk.connectRpc(false, true)` to force a remote RPC connection